Moli/Core/Screens/Profile/ProfileImageAndBannerView.swift

```diff
@@ -0,0 +1,101 @@
+import SwiftUI
+import Mantis
+
+enum ImageType {
+    case profile, banner
+}
+
+struct ProfileImageAndBannerView: View {
+    @EnvironmentObject private var myUser: MyUserDataModel
+    @StateObject var profileImagesDataModel = ProfileImagesDataModel()
```

Comment:
:art: Då den här initieras i denna view så kan vi göra den till en private värde
```suggestion
    @StateObject private var profileImagesDataModel = ProfileImagesDataModel()
```